Sources/Moya/Combine/MoyaPublisher.swift

```diff
@@ -0,0 +1,42 @@
+#if canImport(Combine)
+
+import Combine
+
+// This should be already provided in Combine, but it's not.
+// Ideally we would like to remove it, in favor of a framework-provided solution, ASAP.
+
+@available(OSX 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)
+internal class MoyaPublisher<Output>: Publisher {
```

Comment:
@freak4pc I know you're really busy, but would you mind taking a look at this publisher code? would really appreciate that

Reply:
Hey - I'll only be available in a week and a half (flying to try! Swift tomorrow), not sure if it would help you by then :(

Reply:
Ah no problem, seems to be working fine so I just wanted another pair of eyes on it but it can go to beta and see how it works 👍